AllReadyApp/Web-App/AllReady/ViewModels/Requests/RequestViewModel.cs

```diff
@@ -0,0 +1,18 @@
+namespace AllReady.ViewModels.Requests
+{
+    public class RequestViewModel
```

Comment:
Can you add the following to this view model? If a consumer wants to provide that info, we'll take it. Otherwise, we'll use geocoding to figure it out.
```
public double Latitude { get; set; }
public double Longitude { get; set; }
```